packages/core/src/scp-cf/destination.ts

```diff
@@ -53,6 +51,28 @@ export function parseDestination(destinationJson: DestinationJson | DestinationC
   return sanitizeDestination(destination);
 }
 
+/**
+ * Parses a list of destination configurations and returns a list of SDK compatible destinations.
+ * Only destinations of type 'HTTP' destinations are considered, other types will be skipped.
+ * Configurations of type undefined are assumed to have type 'HTTP'.
+ *
+ * @param destinationJsons - JSON objects as given by the destination service.
+ * @returns A list of SDK compatible destinations.
+ */
+export function parseDestinations(destinationJsons: (DestinationJson | DestinationConfiguration)[]): Destination[] {
```

Comment:
The plural name to `parseDestination` suggests a bit that this does the same just for a list. Perhaps introduce a parseDestinationTypeXYZ() which does a check and use it in the initial loop. `This is easier to understand.

Reply:
Done.